config/webhook/manifests.yaml

```diff
@@ -9,19 +7,13 @@ metadata:
 webhooks:
 - admissionReviewVersions:
   - v1
-  - v1beta1
   clientConfig:
     service:
       name: webhook-service
       namespace: system
       path: /mutate-vertica-com-v1beta1-verticadb
   failurePolicy: Fail
   name: mverticadb.kb.io
-  namespaceSelector:
```

Comment:
Sorry what do you mean by is no longer generated? because with `controller-gen crd webhook ....` it automatically generates the file you see right now. Or do you mean we should still keep the part you highlighted and prevent this file to be re-generated each time?

Reply:
We changed this file to not be generated anymore.  We did this because we needed to add the namespace selector to it in order to get the webhook to work.  So, I think we still need that selector added.